.binder/postBuild

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
- add a README file explaining the rationale: to mimick the same package config as on Colab
- add missing requirements.txt with PySDM-examples